packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -686,7 +686,12 @@ class FlutterPlugin implements Plugin<Project> {
      */
     private void configureLegacyPluginEachProjects(Project project) {
         try {
-            if (!settingsGradleFile(project).text.contains("'.flutter-plugins'")) {
+            // Read the contents of the settings.gradle file.
+            // Remove block/line comments
+            String settingsText = settingsGradleFile(project).text
+            settingsText = settingsText.replaceAll(/(?s)\/\*.*?\*\//, '').replaceAll(/(?m)\/\/.*$/, '')
```

Comment:
I think this regex would fail if nested block comments are nested.

Consider `/*/**/ */` in which only `/*/**/` would be matched.

Reply:
Then again I don't think nested block comments are supported in Groovy, so disregard.

Reply:
@andrewkolos Thank you for your review.